packages/smooth_app/lib/pages/preferences/user_preferences_languages_list.dart

```diff
@@ -0,0 +1,417 @@
+// Flutter Supported Languages List
+// https://api.flutter.dev/flutter/dart-ui/Locale-class.html
+
+List<String> languageNameList = <String>[
+  'Afar',
+  'Afrikaans',
+  'Akan',
+  'Amharic',
+  'Arabic',
+  'Assamese',
+  'Azerbaijani',
+  'Belarusian',
+  'Bulgarian',
+  'Bambara',
+  'Bengali',
+  'Tibetan',
+  'Breton',
+  'Bosnian',
+  'Catalan',
+  'Chechen',
+  'Corsican',
+  'Czech',
+  'Chuvash',
+  'Welsh',
+  'Danish',
+  'German',
+  'Greek',
+  'English',
+  'Esperanto',
+  'Spanish',
+  'Estonian',
+  'Basque',
+  'Persian',
+  'Finnish',
+  'Filipino',
+  'Faroese',
+  'French',
+  'Irish',
+  'Scots Gaelic',
+  'Galician',
+  'Gujarati',
+  'Hausa',
+  'Hebrew',
+  'Hindi',
+  'Croatian',
+  'Haitian Creole',
+  'Hungarian',
+  'Armenian',
+  'Indonesian',
+  'Sichuan Yi',
+  'Icelandic',
+  'Italian',
+  'Inuktitut',
+  'Japanese',
+  'Javanese',
+  'Georgian',
+  'Kazakh',
+  'Khmer',
+  'Kannada',
+  'Korean',
+  'Kurdish',
+  'Cornish',
+  'Kirghiz',
+  'Latin',
+  'Luxembourgish',
+  'Lao',
+  'Lithuanian',
+  'Latvian',
+  'Malagasy',
+  'Maori',
+  'Mongolian',
+  'Marathi',
+  'Malay',
+  'Maltese',
+  'Burmese',
+  'Norwegian Bokmål',
+  'Nepali',
+  'Dutch',
+  'Norwegian Nynorsk',
+  'Norwegian',
+  'South Ndebele',
+  'Occitan',
+  'Panjabi',
+  'Polish',
+  'Portuguese',
+  'Quechua',
+  'Romansh',
+  'Romanian',
+  'Russian',
+  'Sanskrit',
+  'Sardinian',
+  'Sindhi',
+  'Sangro',
+  'Sinhala',
+  'Slovak',
+  'Slovenian',
+  'Shona',
+  'Somali',
+  'Albanian',
+  'Serbian',
+  'Swati',
+  'Southern Sotho',
+  'Swedish',
+  'Swahili',
+  'Tamil',
+  'Telugu',
+  'Tajik',
+  'Thai',
+  'Tigrinya',
+  'Tagalog',
+  'Setswana',
+  'Turkish',
+  'Tsonga',
+  'Tatar',
+  'Twi',
+  'Tahitian',
+  'Uighur',
+  'Ukrainian',
+  'Urdu',
+  'Uzbek',
+  'Venda',
+  'Vietnamese',
+  'Walloon',
+  'Wolof',
+  'Xhosa',
+  'Yiddish',
+  'Yoruba',
+  'Chinese',
+  'Zulu',
+];
+
+Map<String, String> supportedLanguages = <String, String>{
```

Comment:
Prev comment , why use `List<String> languageNameList = <String>[ ` then

Reply:
I realized I can just get all the keys.
Thanks for the suggestion.